server/routes/api.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const Questions = require('../models/questions');
+const db = require('../models/db');
+
+const api = express.Router();
+
+// post question
+api.post('/postQuestion', (req, res) => {
+  if (req.body.question) {
+    const parsedQuestion = JSON.parse(req.body.question);
```

Comment:
sorry forgot to mention this breaks my test, so have removed it locally. I think this is done by the body parser. How are you testing the route locally?

Reply:
ok fair enough. I take it the parse was necessary. Interesting I breaks a request from front end. Let's leave it for now and see how things go.